cmd/kube-apiserver/app/server.go

```diff
@@ -438,6 +440,14 @@ func buildGenericConfig(
 	lastErr error,
 ) {
 	genericConfig = genericapiserver.NewConfig(legacyscheme.Codecs)
+	genericConfig.IsTerminating = func() bool {
+		select {
+		case <-stopCh:
```

Comment:
is `readinessStopCh` better?

https://github.com/kubernetes/kubernetes/blob/53f3699cf936b14e04b8533e51e8e7ffe0eeb7ef/staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go#L166-L168